ppml/trusted-realtime-ml/scala/docker-occlum/Dockerfile

```diff
@@ -88,7 +88,7 @@ RUN cd /opt && \
 FROM occlum/occlum:0.27.0-ubuntu20.04 as ppml
 
 ARG FLINK_VERSION=1.11.3
-ARG BIGDL_VERSION=2.0.1-SNAPSHOT
+ARG BIGDL_VERSION=2.1.0-SNAPSHOT
```

Comment:
duplicated VERSION arguments? why?

Reply:
Updated and delete bigdl home. Please review again.